x/stakeibc/keeper/deposit_records_test.go

```diff
@@ -435,8 +435,8 @@ func (s *KeeperTestSuite) CheckStateAfterStakingDepositRecords(tc DepositRecords
 		val2 := tc.hostZone.Validators[1]
 		totalWeight := val1.Weight + val2.Weight
 
-		val1Delegation := uint64(depositRecord.Amount) * val1.Weight / totalWeight
-		val2Delegation := uint64(depositRecord.Amount) * val2.Weight / totalWeight
+		val1Delegation := depositRecord.Amount.Mul(sdk.NewIntFromUint64(val1.Weight)).Quo(sdk.NewIntFromUint64(totalWeight))
```

Comment:
doesn't have to be in this PR, but do you think we should move to sdk.Ints for all int's in our protos or just amounts?

Reply:
Can't hurt to move everything to ints!

Reply:
https://github.com/Stride-Labs/stride/issues/508